### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mar4enkofff**
https://Mar4enkofff.github.io/layout_html-form/
https://Mar4enkofff.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,6 +12,184 @@
   </head>
   <body>
     <h1>HTML Form</h1>
```
Comment (SanyaBratashchuk) [inline src/index.html:14]:
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -12,6 +12,184 @@
...
+              value="No"
+            >No
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favourite color?
+            <input
+              type="color"
+              id="head"
+              name="head"
```
Comment (SanyaBratashchuk) [inline src/index.html:121]:
```suggestion
              name="color"
```

Comment (SanyaBratashchuk) [reply src/index.html:121]: fix `names`

File: src/index.html [hunk 3]
```diff
@@ -12,6 +12,184 @@
...
+        <div class="form-field">
+          <label for="comments">Comments
+            <textarea autocomplete="off" name="text__field"></textarea>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="final_check">Would you recommend us?
+            <select
+              name="recommend"
+              required>
```
Comment (SanyaBratashchuk) [inline src/index.html:178]:
```suggestion
            <select
              name="recommend"
              required
            >
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Comment by alexandra-protyanova** [comment]
do not close your pr, I can't review it when it closed

**Comment by alexandra-protyanova** [comment]
Снимок экрана 2022-10-17 в 16 27 59
Open pr and click here to ask for review

**Comment by Mar4enkofff** [comment]
how to open it? if the button is not active more?

пн, 17 жовт. 2022 р. о 16:28 Alexandra Protyanova ***@***.***>
пише:

**Comment by Mar4enkofff** [comment]
> Снимок экрана 2022-10-17 в 16 27 59
> 
> Open pr and click here to ask for review

how to open here if the button is not more active?

**Pull request closed without merging**